tests/configs.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+- hosts: all
+
+  vars:
+    manala_sensu_configs_user: root
+    manala_sensu_configs_group: sensu
+    manala_sensu_configs_mode: 640
+
+    manala_sensu_configs:
+      - file: transport.json
+        config:
+          transport:
```

Comment:
I was lazy to write macro to dump json, `{{ item.config|default({})|to_nice_json }}` was so simple :)

I try to avoid any BC in this PR, but I agree that `role.sensu` need to follow manala config structures... let's keep it for next release.